aspnetcore/performance/caching/response.md

```diff
@@ -1,92 +1,72 @@
 ---
 title: Response Caching | Microsoft Docs
-author: ardalis
-description: 
+author: rick-anderson
+description: Explains how to use Response caching to lower bandwidth and increase performance.
 keywords: ASP.NET Core,
 ms.author: riande
 manager: wpickett
 ms.date: 10/14/2016
 ms.topic: article
 ms.assetid: cb42035a-60b0-472e-a614-cb79f443f654
-ms.technology: aspnet
 ms.prod: aspnet-core
 uid: performance/caching/response
 ---
 # Response Caching
 
->[!WARNING]
-> This page documents version 1.0.0-rc1 and has not yet been updated for version 1.0.0
-
-[Steve Smith](http://ardalis.com)
+[Rick Anderson](https://twitter.com/RickAndMSFT) and [Steve Smith](http://ardalis.com)
 
 [View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/performance/caching/response/sample)
 
 ## What is Response Caching
 
-*Response caching* refers to specifying cache-related headers on HTTP responses made by ASP.NET Core MVC actions. These headers specify how you want client and intermediate (proxy) machines to cache responses to certain requests (if at all). This can reduce the number of requests a client or proxy makes to the web server, since future requests for the same action may be served from the client or proxy's cache. In this case, the request is never made to the web server.
-
-![image](response/_static/proxy-and-cache.png)
-
-The primary HTTP header used for caching is `Cache-Control`. The [HTTP 1.1 specification](https://tools.ietf.org/html/rfc7234#section-5.2) details many options for this directive. Three common directives are:
-
-**public**
-
-   Indicates that the response may be cached.
+*Response caching* adds cache-related headers to responses. These headers specify how you want client, intermediate (proxy) machines and middleware to cache responses. Response caching can reduce the number of requests a client or proxy makes to the web server. Response caching can also reduce the amount of work the web server performs to generate the response. Repeated request URLs that match may be served from the cache of the client, proxy or server. Cached responses can be returned by the client, the proxy or served by the response caching middleware. Cached requests served by the client or proxy save bandwidth and server load. Cached requests served by middleware can reduce web app server load.
 
-**private**
+The primary HTTP header used for caching is `Cache-Control`. See the [HTTP 1.1 Caching](https://tools.ietf.org/html/rfc7234#section-5.2) and [Cache-Control](https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.9) for more information. Common cache directives:
```

Comment:
Just curious why we are adding links to both resources? From a quick read, they seem to have very similar information, which is why I assume the original doc only linked to one resource.